Springboot stack 0.3.23 not compatiable with older versions.

Running Springboot app that is using 

*******************************************
<parent><!--required parent POM-->
    <groupId>dev.appsody</groupId>
    <artifactId>spring-boot2-stack</artifactId>
    <version>[0.3, 0.4)</version>
    <relativePath/>
  </parent>
*************************************************

When  a webhook deploy runs this is the error that is seen:

*****************************
[Buildah] [ERROR] [ERROR] Some problems were encountered while processing the POMs:
[Buildah] [FATAL] Non-resolvable parent POM dev.appsody:spring-boot2-stack:[0.3, 0.4) for dev.appsody:application:0.0.1-SNAPSHOT: No versions matched the requested parent version range '[0.3, 0.4)' and 'parent.relativePath' points at no local POM @ line 6, column 11
[Buildah]  @ 

************************

To get it to work we had to use 
****************
<parent><!--required parent POM-->
    <groupId>dev.appsody</groupId>
    <artifactId>spring-boot2-stack-snowdrop</artifactId>
    <version>[0.3, 0.4)</version>
    <relativePath/>
  </parent>

********************

According to discussions the version field needs to be updated to just 0.4 so the correct error is displayed.

--- a/incubator/java-spring-boot2/stack.yaml
+++ b/incubator/java-spring-boot2/stack.yaml
@@ -1,5 +1,5 @@
 name: Spring Boot®
-version: 0.3.23
+version: 0.3.24
 description: Spring Boot using OpenJ9 and Maven
 license: Apache-2.0
 language: java
@@ -18,6 +18,6 @@ templating-data:
   baseimage: kabanero/ubi8-maven:0.3.0
   finalimage: adoptopenjdk/openjdk8-openj9:ubi-jre
   parentpomgroup: dev.appsody
-  parentpomid: spring-boot2-stack-snowdrop
+  parentpomid: spring-boot2-stack
   parentpomrange: '[0.3, 0.4)'
   parentpomfilename: appsody-boot2-snowdrop-pom.xml

